insights/specs/datasources/httpd.py

```diff
@@ -92,11 +98,17 @@ def get_httpd_configuration_files(httpd_root):
             server_root = httpd_root
             # Add it only when it exists
             all_paths.add(main_httpd_conf)
+            section_number = 0
             for line in cfp.readlines():
-                if line.strip().startswith("ServerRoot"):
+                line = line.strip()
+                if line.startswith("<") and not line.startswith("</"):
+                    section_number = section_number + 1
+                if line.startswith("</"):
```

Comment:
As this second `if` is the another case of the above one, using `elif` here could be more suitable than `if`.